config/config.default.yaml

```
@@ -565,6 +565,19 @@ sector:
     rural:
     - air
     - ground
+  residential_heat:
+    dsm:
+      enable: false
+      direction: "overheat-undercool"
```

```suggestion
      direction:
      - overheat
      - undercool
```
could avoid encoding that as string (implementation already works for this, just the documentation would need to be adjusted).